src/components/startPage/main.css

```diff
@@ -0,0 +1,115 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: "roboto-slab,non-serif";
+    font-size: 1vh;
```

Comment:
As for responsiveness, I recommend px and adding media querries. In order not to mess with font size on different screens, you can check something like clamp.